indico/modules/logs/controllers.py

```diff
@@ -159,3 +161,34 @@ def object(self):
     @property
     def object_tzinfo(self):
         return get_timezone(config.DEFAULT_TIMEZONE)
+
+
+class RHResendEmail(RHManageEventBase):
+    """Resend an email log entry."""
+
+    def _process_args(self):
+        self.entry = EventLogEntry.get(request.view_args['log_entry_id'])
```

Comment:
Nothing here checks that it belongs to the same event! You need to give the EventLogEntry a locator and setup normalization on this RH:

```py
    normalize_url_spec = {
        'locators': {
            lambda self: self.entry
        }
    }
```